scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
As we don't have a support for cross building docker images for other architecture, so this will not create any issues I feel.